EducationManagement/Services/Implementations/UserService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using EducationManagement.Services.Abstractions;
+using EM.Database;
+using System;
+using System.Linq;
+using System.Text;
+
+namespace EducationManagement.Services.Implementations
+{
+    public class UserService : IUserService
+    {
+        private readonly DataContext db = new DataContext();
+
+        public void UpdateAvatar(string token, string url)
+        {
+            string base64Decoded;
+
+            byte[] data = Convert.FromBase64String(token);
+
+            base64Decoded = ASCIIEncoding.ASCII.GetString(data);
+
+            string[] temp = base64Decoded.Split('@');
+
+            var userId = Int32.Parse(temp[0]);
```

Comment:
you can extract above codes to a function for using in future

Reply:
It's reasonable.